src/tools/loadbal.py

```diff
@@ -0,0 +1,517 @@
+#  Copyright (C) 2017(Dr. Horacio Andres Vargas Guzman)
```

Comment:
please run ```autopep8``` tool on this file, it is quite difficult to read the current formatting.

Reply:
Done